framework/base/ErrorHandler.php

```diff
@@ -196,7 +196,7 @@ public function handleHhvmError($code, $message, $file, $line, $context, $backtr
      * @param string $message the error message.
      * @param string $file the filename that the error was raised in.
      * @param int $line the line number the error was raised at.
-     * @return bool whether the normal error handler continues.
+     * @return false whether the normal error handler continues.
```

Comment:
I assume this is done to prevent malfunctioning after future changes. The return value is important for set_error_handler https://github.com/yiisoft/yii2/blob/master/framework/base/ErrorHandler.php#L65. 

According to http://php.net/manual/en/function.set-error-handler.php, `If the function returns FALSE then the normal error handler continues.` When the behavior of this custom errorHandler will change (for whatever reason in the future) and return a true value, the HHVM logic will still function correctly.

However, I would vote to cover this in tests and apply your suggestions.